scripts/check-gmp.gmp

```diff
@@ -1090,7 +1090,10 @@ def to_int(
     :param default: If the value is missing in the dict use this default
 
     """
-    value = source_dict.get(key, default)
+
+    value = source_dict.get(key)
+    if value in [None, ""]:
```

Comment:
Can be simplified to:
```suggestion
    if not value:
```

Reply:
This change the behavior, because it will match in case value = 0.

Reply:
ah didn't know zero should be excluded here.